[INFO] [launch]: All log files can be found below /root/.ros/log/2025-05-30-13-26-37-674820-49a9edf06ace-21919
[INFO] [launch]: Default logging verbosity is set to INFO
architecture_type                   := awf/universe
autoware_launch_file                := planning_simulator.launch.xml
autoware_launch_package             := autoware_launch
consider_acceleration_by_road_slope := False
consider_pose_by_road_slope         := True
global_frame_rate                   := 30.0
global_real_time_factor             := 1.0
global_timeout                      := 180
initialize_duration                 := 30
launch_autoware                     := True
launch_rviz                         := false
output_directory                    := /autoware_map
port                                := 5555
record                              := false
rviz_config                         := 
scenario                            := /autoware_map/generated_scenarios/scenario_2.yaml
sensor_model                        := sample_sensor_kit
sigterm_timeout                     := 8
use_sim_time                        := False
vehicle_model                       := sample_vehicle
[INFO] [scenario_test_runner.py-1]: process started with pid [21920]
[INFO] [simple_sensor_simulator_node-2]: process started with pid [21922]
[INFO] [openscenario_interpreter_node-3]: process started with pid [21924]
[INFO] [openscenario_preprocessor_node-4]: process started with pid [21926]
[INFO] [visualization_node-5]: process started with pid [21928]
[scenario_test_runner.py-1] [INFO] [1748611598.245747686] [simulation.scenario_test_runner]: connection established with preprocessor
[scenario_test_runner.py-1] [INFO] [1748611599.695397781] [simulation.scenario_test_runner]: Result of /simulation/openscenario_preprocessor/load: success
[scenario_test_runner.py-1] [INFO] [1748611599.696042924] [simulation.scenario_test_runner]: Result of /simulation/openscenario_preprocessor/check: True
[scenario_test_runner.py-1] [INFO] [1748611599.696756510] [simulation.scenario_test_runner]: derived : /autoware_map/scenario_test_runner/scenario_2/scenario_2.xosc
[scenario_test_runner.py-1] [INFO] [1748611599.697317878] [simulation.scenario_test_runner]: Result of /simulation/openscenario_preprocessor/check: False
[scenario_test_runner.py-1] [INFO] [1748611599.697594034] [simulation.scenario_test_runner]: finish derivation
[scenario_test_runner.py-1] [INFO] [1748611599.697865026] [simulation.scenario_test_runner]: /autoware_map/scenario_test_runner/scenario_2/scenario_2.xosc
[scenario_test_runner.py-1] [INFO] [1748611599.698214035] [simulation.scenario_test_runner]: Run scenario_2.xosc (1 of 1)
